src/main/java/frc/team2412/robot/commands/arm/SetArmCommand.java

```diff
@@ -1,31 +1,37 @@
 package frc.team2412.robot.commands.arm;
 
+import static frc.team2412.robot.subsystems.ArmSubsystem.ArmConstants.PositionType.*;
+
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import frc.team2412.robot.subsystems.ArmSubsystem;
+import frc.team2412.robot.subsystems.ArmSubsystem.ArmConstants.PositionType;
 
 public class SetArmCommand extends CommandBase {
 
 	private ArmSubsystem armSubsystem;
-	private double targetAngle;
+	private PositionType positionType;
 
-	public SetArmCommand(ArmSubsystem armSubsystem, double targetAngle) {
+	public SetArmCommand(ArmSubsystem armSubsystem, PositionType positionType) {
 		this.armSubsystem = armSubsystem;
-		this.targetAngle = targetAngle;
+		this.positionType = positionType;
 		addRequirements(armSubsystem);
 	}
 
 	@Override
-	public void execute() {
-		armSubsystem.rotateArmTo(targetAngle);
+	public void initialize() {
+		armSubsystem.setPosition(positionType);
+		armSubsystem.setArmGoal(positionType.armAngle);
+		// Prepares wrist to score whenever position changes to save time
+		armSubsystem.setWristGoal(positionType.prescoringWristAngle);
```

Comment:
Seems like this will want to be at the wrist score position to pickup a cone from the substation.
We would seemingly also want wrist retracted when going into low position.